lib/course_planner/permissions.ex

```diff
@@ -7,9 +7,12 @@ defimpl Canada.Can, for: CoursePlanner.User do
     PageController,
     ScheduleController,
     TaskController,
-    User,
```

Comment:
no objection.
it's just on my nerve ( speaking of being a c++ programmer . hhahahha)